src/main/kotlin/no/nav/familie/ef/sak/api/gui/BrevMellomlagerController.kt

```diff
@@ -0,0 +1,44 @@
+package no.nav.familie.ef.sak.api.gui
+
+import no.nav.familie.ef.sak.api.dto.MellomlagretBrevDto
+import no.nav.familie.ef.sak.repository.domain.MellomlagretBrev
+import no.nav.familie.ef.sak.service.MellomlagringBrevService
+import no.nav.familie.ef.sak.service.TilgangService
+import no.nav.familie.kontrakter.felles.Ressurs
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+@RequestMapping(path = ["/api/brev/mellomlager/"])
+@ProtectedWithClaims(issuer = "azuread")
+class BrevMellomlagerController(private val tilgangService: TilgangService,
+                                private val mellomlagringBrevService: MellomlagringBrevService) {
+
+    @PostMapping("/{behandlingId}")
+    fun mellomlagreBrevverdier(@PathVariable behandlingId: UUID,
+                               @RequestBody mellomlagretBrev: MellomlagretBrevDto): Ressurs<UUID> {
+        tilgangService.validerTilgangTilBehandling(behandlingId)
+
+        return Ressurs.success(mellomlagringBrevService.mellomLagreBrev(MellomlagretBrev(behandlingId,
+                                                                                         mellomlagretBrev.brevverdier,
+                                                                                         mellomlagretBrev.brevmal,
+                                                                                         mellomlagretBrev.versjon)))
+    }
+
+
+    @GetMapping("/{behandlingId}/{brevmal}")
```

Comment:
Her tolker jeg det som att man må velge brevmal i frontend først, før man henter mellomlagring. 
Hadde det vært smidigere hvis frontend kun hentet mellomlagret brev, og då oppdaterte frontend med hvilken mal som blir brukt? Eller hva er behovet for å først velge brevmal og sen hente mellomlagring?

Reply:
Jeg tenker at saksbehandler uansett må kunne "overstyre" hvilken brevmal de skal bruke. Men kan vurdere om vi default skal sette den til den mellomlagrede malen.